Framework/Backend/websocket/server.js

```diff
@@ -133,6 +133,42 @@ class WebSocket {
     client.on('error', (err) => this.logger.error(`Connection ${err.code}`));
   }
 
+  /**
+   * Make criteria more readable.
+   * This code is a close copy of InfoLogger/public/logFilter/LogFilter.js LN 101 toObject()
+   * @param {object} criteria - criteria to be minified
+   * @returns {object} minimal filter object
+   */
+  minifyCriteria(criteria) {
```

Comment:
@Houwie7000  as this is a utility function, I think it is better if we move it to an independent JS module in an utility folder.
As currently there is no such folder, please create one and add this function to its own module. Then call it from there in the WS server.
The tests associated to this function should match the folder structure as well